sdk/servicebus/azure-servicebus/dev_requirements.txt

```diff
@@ -1,2 +1,2 @@
 -e ../../../tools/azure-sdk-tools
--e ../../core/azure-common
\ No newline at end of file
+-e ../../core/azure-common
```

Comment:
This means this can be removed

Reply:
100%. I'm just running local test on `azure-servicemanagement-legacy` before pushing the next commit with both those removed :)

Reply:
(the only other one that's using `azure-common` is legacy)